### PR TITLE
Add "Leave Room" button to VR HUD

### PR DESCRIPTION
**Pull request by gfodor**
Adds the "Leave Room" UX from 2d mode into VR, and also re-arranges the HUD in VR a bit.

### CODE REVIEW

File: src/components/leave-room-button.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+AFRAME.registerComponent("leave-room-button", {
+  init() {
+    this.onClick = () => {
+      console.log("click");
```
Comment (brianpeiris) [inline src/components/leave-room-button.js]: Remove debug log

File: src/components/leave-room-button.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      this.el.sceneEl.emit("leave_room_requested");
```
Comment (brianpeiris) [inline src/components/leave-room-button.js]: nit: We seem to alternate between using hyphens and underscores in our event names. Would be nice to settle on one of them. Looks like underscores are winning, so just something to keep an eye out for in other reviews.

File: src/components/leave-room-button.js (hunk 1 above)
Comment (gfodor) [reply src/components/leave-room-button.js]: oops, nice catch

File: src/components/leave-room-button.js (hunk 2 above)
Comment (gfodor) [reply src/components/leave-room-button.js]: ah yeah i've noticed this too

**Pull request merged**